### ISSUE TITLE
How to evaluate the performance of the proposed method on the KITTI dataset？

### ISSUE THREAD

**Issue by zhangyanan0306** [hzxie/GRNet#7]
when I run: python3 runner.py --test --weights=GRNet-KITTI.pth
![QQ截图20200714194842](https://user-images.githubusercontent.com/49965736/87526742-cc7bb780-c6bd-11ea-8605-be9d2114b323.png)
The above error occurred.
So，how do I test GRNet on the KITTI dataset?
How can I get the Consistency and Uniformity values described in your paper? 
Does your open source code contain the corresponding code? Where can I find it?

**Comment by hzxie** [comment]
The above error occurred due to the lack of ground truth of 3D objects.

The code for Uniformity can be found in https://github.com/liruihui/PU-GAN.
The code for Consistency can be implemented by yourself. It is easy, isn't it?

You can generate the prediction for KITTI with the following commands:

```
python runner.py --inference --weight /path/to/pretrained/model
```

**Comment by zhangyanan0306** [comment]
> The above error occurred due to the lack of ground truth of 3D objects.
> 
> The code for Uniformity can be found in https://github.com/liruihui/PU-GAN.
> The code for Consistency can be implemented by yourself. It is easy, isn't it?
> 
> You can generate the prediction for KITTI with the following commands:
> 
> ```
> python runner.py --inference --weights /path/to/pretrained/model
> ```

OK, thank you very much！

**Comment by zhangmozhe** [comment]
Hi @zhangyanan0306 Did you successfully run the uniformity evaluation? The code (https://github.com/liruihui/PU-GAN) requires a mesh input, but now we only have point cloud results as well as point gt.

**Comment by hzxie** [comment]
@zhangmozhe 
Please refer to https://github.com/liruihui/PU-GAN/issues/18

**Comment by zhangmozhe** [comment]
> @zhangmozhe
> Please refer to [liruihui/PU-GAN#18](https://github.com/liruihui/PU-GAN/issues/18)

Their code requires mesh as input. Then how to compute the metric if we only have the point cloud ground truth? Thanks.

**Comment by hzxie** [comment]
The code for uniformity is available in #20.
